vizro-core/hatch.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
 ]
 
 [envs.default.env-vars]
-DASH_DEBUG = "true"
+# DASH_DEBUG = "true"
```

Comment:
Did you see any weird behaviours with the caching when `DASH_DEBUG = "true"`?

Reply:
I see more cache objects created in DASH_DEBUG mode